src/arqa.py

```diff
@@ -117,6 +118,112 @@ def arc_fit_qa(slf, fit, outfil=None, ids_only=False, title=None):
     return
 
 
+def flexure(slf, det, flex_dict):
+    """ QA on flexure measurement
+
+    Parameters
+    ----------
+    slf
+    det
+    flex_dict
+
+    Returns
+    -------
+
+    """
+    # Setup
+    nobj = len(slf._specobjs[det-1])
+    if nobj == 0:
+        return
+    ncol = min(3,nobj)
+    nrow = nobj // ncol + ((nobj%ncol) > 0)
+
+
+    plt.figure(figsize=(8, 5.0))
+    plt.clf()
+    gs = gridspec.GridSpec(nrow, ncol)
+
+    # Correlation QA
+    for o,specobj in enumerate(slf._specobjs[det-1]):
+        ax = plt.subplot(gs[o//ncol, o%ncol])
+        # Fit
+        fit = flex_dict['polyfit'][o]
+        xval = np.linspace(-10., 10, 100) + flex_dict['corr_cen'][o] + flex_dict['shift'][o]
+        model = (fit[2]*(xval**2.))+(fit[1]*xval)+fit[0]
+        mxmod = np.max(model)
+        ylim = [np.min(model/mxmod), 1.5]
```

Comment:
On the QA, 1.5 seems a little too generous. Maybe 1.1 would be better to remove the "extra" whitespace?